Samples/CayenneDecoderTest/CayenneTest.cs

```diff
@@ -284,11 +280,10 @@ public void GPSLocation()
             cayenneEncoder.AddGPS(channelgps, lat, lon, alt);
             var buff = cayenneEncoder.GetBuffer();
             CayenneDecoder cayenneDecoder = new CayenneDecoder(buff);
-            cayenneDecoder.TryGetGPSLocation(out GPSLocation gPSLocation);
-            Assert.Equal(gPSLocation.Latitude, lat);
-            Assert.Equal(gPSLocation.Longitude, lon);
-            Assert.Equal(gPSLocation.Altitude, alt);
-            Assert.Equal(gPSLocation.Channel, channelgps);
+            Assert.Equal(cayenneDecoder.cayenneDevice.GPSLocation[0].Latitude, lat);
+            Assert.Equal(cayenneDecoder.cayenneDevice.GPSLocation[0].Longitude, lon);
+            Assert.Equal(cayenneDecoder.cayenneDevice.GPSLocation[0].Altitude, alt);
+            Assert.Equal(cayenneDecoder.cayenneDevice.GPSLocation[0].Channel, channelgps);
         }
     }
 }
```

Comment:
Could you please add a e2e test with multiple values of same type?

Reply:
I can try to before end of the week.